src/cli.ts

```diff
@@ -1,8 +1,12 @@
 import { startPatch } from '.'
 
-if (!process.argv[2]) {
-  console.error('$ pnpm-patch-i <package-name>')
+const args = process.argv.slice(2)
+const name = args[args.length === 1 ? 0 : 1]
+const ignoreExisting = args[0] === '--ignore-existing'
```

Comment:
I think maybe we should forward all the arguments to it.

Reply:
I hesitated between the 2 approaches, I'll update the PR.